Java/Primitive Data Type/Floating Types/README.md

```diff
@@ -1 +1,112 @@
+4. **MAX_EXPONENT**: Maximum exponent a finite double variable may have.
+5. **MIN_EXPONENT**: Minumum exponent a normalized double variable may have.
+6. **BYTES**: The number of bytes used to represent a double value.
+7. **SIZE**: The number of bits used to represent a double value.
+8. **TYPE**: The Class instance representing the primitive type double.
+9. **NaN**: A constant holding a Not-a-Number (NaN) value of type double.
+10. **POSITIVE_INFINITY**: A constant holding the negative infinity of type double. 
+11. **NEGATIVE_INFINITY**: A constant holding the negative infinity of type double. 
+
+### **<ins>Constructor</ins>**
+1. **Double(double value)**: Constructs a newly allocated Double object that represents the argument converted to type double.
+2. **Double(String s)**: Constructs a newly allocated Double object that represents the floating-point value of type double represented by the string.
+
+### **<ins>Methods</ins>**
+  1. **byteValue()**: Returns the value of Double as a byte.
+  2. **shortValue()**: Returns the value of Double as a short
+  3. **intValue()**: Returns the value of Double as a int.
+  4. **longValue()**: Returns the value of Double as a long.
+  5. **doubleValue()**: Returns the value of Double as a double.
+  6. **floatValue()**: Returns the value of Double as a float.
+  7. **hashCode()**: Returns the value of Double as a hashcode.
+  8. **isFinite()**: Returns true if the Double value is a finite floating-point value.
+  9. **isInfinite()**: Returns true if the Double value is not a finite floating-point value.
+  10. **isNan()**: Returns true if the Double value is a Not-a-Number (NaN).
+  11. **toString()**: Returns the string corresponding to the double value. 
+  12. **parseDouble()**: Returns a new double initialized to the value represented by the specified String.
+  13. **valueOf()**: Returns the Double object initialized with the double value provided.
+  14. **toHexString()**: Returns the hexadecimal representation corresponding to the double value.
+  15. **doubleToLongBits()**: Returns a representation of the specified floating-point value according to the IEEE 754 floating-point "double format" bit layout.
+  16. **doubleToRawLongBits()**: Returns a representation of the specified floating-point value according to the IEEE 754 floating-point "double format" bit layout, preserving Not-a-Number (NaN) values.
+  17. **LongBitsToDouble()**: Returns the double value corresponding to a given long bit representation.
+  18. **equals()**: Compares the Double object against another Double object.
+  19. **compare()**: Compares the two specified double values.
+  20. **compareTo()**: Compares two Double objects numerically.
+  21. **min()**: Returns the minimuim of the two double values entered.
+  22. **max()**: Returns the maximum of the two double values entered.
+  23. **sum()**: Adds two double values together as per the + operator.
+
+[Back to top](#floating-types)
```

Comment:
Please check the plagiarism of the content!

Reply:
I'll look into it. I copied the class definitions from oracle docs because I didn't want to write incorrect information. I'll edit it and make sure its plagiarism free.

Reply:
I edited the documentation and made it plagiarism free. Let me know if I need to make some other changes.
![Screenshot_1](https://user-images.githubusercontent.com/84243683/134431287-cfbb5499-6da7-4b0b-911c-bacf56168148.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/84243683/134431296-6a141bba-3d66-4580-ba6c-6965e729d130.jpg)

Reply:
Plagiarism is not there, but please correct the spellings